source/agora/node/admin/QRCodeInterface.d

```diff
@@ -19,6 +19,7 @@ import agora.common.Types;
 import agora.consensus.data.PreImageInfo;
 import agora.crypto.Hash;
 import agora.crypto.Key;
+import agora.crypto.Schnorr: Signature;
```

Comment:
Same question, is this used ?

Reply:
https://github.com/bpfkorea/agora/blob/1efa307d45090915d40ba52ef124affafd398a45/source/agora/node/admin/QRCodeInterface.d#L231